atm.js

```diff
@@ -1,21 +1,54 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
+//Checking account
+//Define players
+$(document).ready(function() {
```

Comment:
this is not wrong, but `document.ready` is unecessary here because your script tag is just before the closing `body` tag, and the dom will have loaded by the time the script loads.